src/test/java/HomepageTest.java

```diff
@@ -1,45 +1,146 @@
-import org.assertj.core.api.Assertions;
-import org.openqa.selenium.By;
+import org.assertj.core.api.SoftAssertions;
 import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.chrome.ChromeDriver;
 import org.testng.annotations.AfterClass;
 import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 import pages.Homepage;
 import utils.WebDriverUtils;
 
-import java.time.Duration;
-
-import static org.assertj.core.api.Assertions.assertThat;
-
 public class HomepageTest {
 
     private WebDriver driver;
 
+    private static final String WHITE_COLOR = "#ffffff";
```

Comment:
As it was mention few times, static fields should be declared first before non static fields.
So both colors should be defined before  private WebDriver driver;